projects/demo/src/modules/components/dialog/examples/10/index.less

```diff
@@ -0,0 +1,51 @@
+tui-dialog[data-appearance='ios-style'][data-size] {
+    margin: 1.25rem auto;
+    border: none;
+    border-radius: 0;
+
+    .t-content {
```

Comment:
Please add an example that does not rely on internal structure.

Reply:
fix